src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerUpdateValidator.java

```diff
@@ -86,15 +88,27 @@ public void validateChangeDate(HttpServletRequest request, List<ApiError> errorL
 
     @Override
     public void validateOptionalDtoFields(HttpServletRequest request, List<ApiError> errorList, OfficerFilingDto dto) {
-        Boolean nameHasBeenUpdated = (dto.getNameHasBeenUpdated() == null || dto.getNameHasBeenUpdated());
-        Boolean anyNameFieldsExistInDto = (dto.getTitle() != null || dto.getFirstName() != null || dto.getLastName() != null || dto.getMiddleNames() != null);
+        // For each data section, validation will be performed if either:
+        // - the hasBeenUpdated boolean is true
+        // - the hasBeenUpdated boolean is null and any data is present
 
-        if (nameHasBeenUpdated && anyNameFieldsExistInDto) {
+        final boolean anyNameFieldsExistInDto = (dto.getTitle() != null || dto.getFirstName() != null || dto.getLastName() != null || dto.getMiddleNames() != null);
+        final boolean nameHasBeenUpdated = (dto.getNameHasBeenUpdated() == null && anyNameFieldsExistInDto) || (dto.getNameHasBeenUpdated() != null && dto.getNameHasBeenUpdated());
+        if (nameHasBeenUpdated) {
             validateTitle(request, errorList, dto);
             validateFirstName(request, errorList, dto);
             validateLastName(request, errorList, dto);
             validateMiddleNames(request, errorList, dto);
         }
+
+        final boolean anyNationalityFieldsExistInDto = (dto.getNationality1() != null || dto.getNationality2() != null || dto.getNationality3() != null);
+        final boolean nationalityHasBeenUpdated = (dto.getNationalityHasBeenUpdated() == null && anyNationalityFieldsExistInDto) || (dto.getNationalityHasBeenUpdated() != null && dto.getNationalityHasBeenUpdated());
+        if (nationalityHasBeenUpdated) {
+            validateNationality1(request, errorList, dto);
+            validateNationality2(request, errorList, dto);
+            validateNationality3(request, errorList, dto);
```

Comment:
I've a separate query on Nationality3 validation. Can only be present if Nationality2 is present? Happy to do that separate from this PR.